docs/event.md

```diff
@@ -0,0 +1,93 @@
+## Notes
+
+- It's JSON.
+- In this model there is no hierarchy (cf. o-tracking proposal), and only a couple of mandated fields.
+- If certain properties are present in the payload they will enriched by Spoor and appended to the payload.
+- It's for use by server and client-side systems sending events to Spoor.
+- I've tried to keep terminology human understable - Eg, the removal of 'category', 'key', 'value', 'data' - as these should all be inherent in the data structure etc. Anything bespoke goes in 'meta'.
+- It is intentionally abstracted away from any particular database format and meant to represent a logical, easy to understand data structure.
+- It assumes the consumers of the event stream can manipulate the data in to their desired storage format.
+- It doesn't assume users of the data will need to join the data across tables to find the information they want (a downside of the parent-child relationship). Though consumers may wish to split this data in to their own tables for query optimisation etc.
+
+## Spec 
+
+`event.source` and `event.api-key` are MANDATORY, everything else is OPTIONAL
+
+This allows a low barrier to entry for anyone wanting to submit events to Spoor.
+
+## Enrichments 
+
+Each payload will be scanned for the presence of these properties.
+
+If present the properties will be decorated and/or tokenised. 
+
+To date we've discussed :-
+
+- If `user.session.token` is included it will be validated against the session api and expanded in to a membership uuid.
+- If `content.uuid` is included it will be expanded in to a set of content api meta-data.
+- ...
+
+## Format
+
+```
+{
+    "event": {
+        "source": "email.simple-email-service",  				// Name of the sender's system [1]
+        "type": "opened",  										// The type of event that happened, Eg. 'pageview', 'dwell', 'play' etc.
+		"api-key":	"0f7464b4-3f4d-11e4-984b-00144feabdc0"		// Sender-specific key [6]
+    },
+    "user": {													// Who generate the event
+        "user-agent": "Mozilla ...",
+        "ip": "123.123.123.123",
```

Comment:
If you're offering to pull the user session token from the Cookie header, would the same be done for UA and IP if set, or do you want to mandate that these are explicitly set?